Firmware/ultralcd.cpp

```diff
@@ -6683,12 +6749,6 @@ static void lcd_main_menu()
 
 }
 
-void stack_error() {
-	Sound_MakeCustom(1000,0,true);
-	lcd_display_message_fullscreen_P(_i("Error - static memory has been overwritten"));////MSG_STACK_ERROR c=20 r=4
```

Comment:
@wavexx Can you please also remove the message in the `lang_en*.txt` files as it has been found during compiling in the `not_tran*txt/not_used*.txt`?